packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/FIAPPaymentQueueDeleteTests.m

```diff
@@ -2,10 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#import <OCMock/OCMock.h>
 #import <XCTest/XCTest.h>
 #import "FIAObjectTranslator.h"
 #import "FIAPaymentQueueHandler.h"
+#import "Mocks.h"
 #import "Stubs.h"
```

Comment:
combine (remove mocks) and can you also add a comment to remove subclasses in favor of protocols?